src/editor/Core/Filter.re

```diff
@@ -0,0 +1,29 @@
+/**
+   Filter
+
+   Module to handle filtering of items using various strategies
+ */
+
+let menu = (query, items) =>
+  Types.UiMenu.(
```

Comment:
Is this a place where we could use @CrossR 's reasonFuzz library? https://github.com/CrossR/reasonFuzz

Reply:
@bryphe I separated that with exactly that in mind but would rather leave hooking it up to @CrossR this is just place holder rudimentary filtering logic